SPEC.md

```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Field `acName`
```

Comment:
I guess I am unclear what you are trying to point out with these "field" references - in the case of acKind and acVersion it is a simple 1:1 between the type and the field name, but in the case of AC Name it's a more generic type used by a whole bunch of fields...